src/core/log.js

```diff
@@ -264,6 +302,48 @@ async function updateCallLog({ platform, userId, incomingData }) {
                     authHeader = `Basic ${basicAuth}`;
                     break;
             }
+            // Compose updated call log details centrally
+            const logFormat = getLogFormatType(platform);
+
+            // Get existing log details first (for updates we need to compose on top of existing content)
+            let existingBody = '';
+            try {
+                const getLogResult = await platformModule.getCallLog({
```

Comment:
I see the new solution request "get call log" twice. One is here. Another is on updateCallLog. Any way to avoid duplicated request for same API.

Reply:
We will use another PR to fix it.